### DISCUSSION TITLE
v1.0.0

### DISCUSSION THREAD

**Discussion by imdevarsh** [imdevarsh/meowclient#14] [Announcements]
This adds studios and is ready for use!
I forgot to publish `v0.6.0` to npm so sorry! 😢 

## What's Changed
* Studios by @webdev03 in https://github.com/webdev03/meowclient/pull/12

**Full Changelog**: https://github.com/webdev03/meowclient/compare/v0.6.0...v1.0.0

<hr /><em>This discussion was created from the release <a href='https://github.com/webdev03/meowclient/releases/tag/v1.0.0'>v1.0.0</a>.</em>

**Comment by hello-smile6** [comment]
Nice! (Could you publish isScratcher to the NPM registry for me? )

**Reply by imdevarsh** [reply]
Why don't you?

**Reply by hello-smile6** [reply]
> Why don't you?

Don't want to share my email address.

**Reply by imdevarsh** [reply]
Just use `eml.monster` or make an alt email.

**Reply by hello-smile6** [reply]
> Just use `eml.monster` or make an alt email.

I have an alt one (I've even shared it on Scratch). Should I use that? I'm still worried about unsolicited email, though.

**Reply by imdevarsh** [reply]
eml.monster hides real email.. you could connect your alt to eml.monster for ultra protection

**Reply by imdevarsh** [reply]
It also has digest option

**Reply by hello-smile6** [reply]
Okay, I will use that. I used 9p as username.

**Reply by hello-smile6** [reply]
@webdev03 How do I publish to npm?

**Reply by hello-smile6** [reply]
@webdev03 It won't let me publish!

**Reply by imdevarsh** [reply]
What happened?

**Reply by hello-smile6** [reply]
> What happened?

It published
